### PR TITLE
Add Prereq

### PR DESCRIPTION
**Pull request by PFCKrutonium**
Adds List of Prerequisites

### CODE REVIEW

**Comment by PFCKrutonium** [comment]
I *think* I am done with this, unless you want me to change anything else.

File: docs/prerequisites.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+Must Have:  
+[Node.js x64](https://nodejs.org/en/)  
+[Python 2.7](https://www.python.org/downloads/)  
+[Visual Studio 2013 or 2015](https://www.visualstudio.com/en-us/products/visual-studio-community-vs.aspx)  
+
+Should Have:
+
+[Git](https://git-scm.com/download/win)
+
+##OS X
```
Comment (bbondy) [inline docs/prerequisites.md]: Please only list Windows for now since there is no information below that is useful.

File: docs/prerequisites.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
+#Prerequisites
+
+##Windows
+
+Ensure you have the following installed:  
+Must Have:
```
Comment (bbondy) [inline docs/prerequisites.md]: nit:  You can remove these 2 lines

File: docs/prerequisites.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+
+Ensure you have the following installed:  
+Must Have:  
+[Node.js x64](https://nodejs.org/en/)  
+[Python 2.7](https://www.python.org/downloads/)  
+[Visual Studio 2013 or 2015](https://www.visualstudio.com/en-us/products/visual-studio-community-vs.aspx)  
+
+Should Have:
+
+[Git](https://git-scm.com/download/win)
```
Comment (bbondy) [inline docs/prerequisites.md]: Git and node are pretty much always implied.  I'm fine with keeping them though but please make the whole list point form and remove the text `Should Have:`

File: README.md [hunk 4]
```diff
@@ -17,11 +19,11 @@ Laptop and Desktop browser for OS X, Windows and Linux.
...
 
-2. Open the working directory:
+3. Open the working directory:
 
         cd browser-laptop
 
-3. Install the Node (v5+) dependencies:
+4. Install the Node (v5+) dependencies:
 
         npm install
```
Comment (bbondy) [inline README.md]: Please have a new section under installation that has the text of your #1 and you can change the Installation section back to normal.

**Comment by bbondy** [comment]
Please squash all into one commit with `git rebase -i master`

**Comment by PFCKrutonium** [comment]
kk

**Comment by PFCKrutonium** [comment]
I have no idea how to use Vim, and googling offers nothing of help :X

**Comment by bbondy** [comment]
`git config core.editor notepad`

**Comment by PFCKrutonium** [comment]
Notepad is also useless for this because of the wrong type of line ending... :/

**Comment by bbondy** [comment]
You could do notepad++ which will handle that better I bet or something else.

**Comment by PFCKrutonium** [comment]
So I have it working with Notepad++, but I am not sure about how to actually tell it to squash. I've tried selecting the first commit and squashing everything, that didn't work. I've tried selecting the last commit and squashing everything, that didn't work but did tell me something about "allow-empty"... Help?

**Comment by bbondy** [comment]
Check here, I think you want to keep the first and squash the others.
http://gitready.com/advanced/2009/02/10/squashing-commits-with-rebase.html

**Comment by PFCKrutonium** [comment]
Finally! :D Did it!

**Comment by bbondy** [comment]
\o/ Thanks for the cleanup and for helping future people get setup!

**Comment by PFCKrutonium** [comment]
No problem, I just wish Vim wasn't so damn confusing... nano plz.

**Pull request merged**